builder/context/native.go

```diff
@@ -40,11 +40,9 @@ var nativeCommandsForLang = map[string]map[string][]string{
 		},
 		"typescript": {
 			"npm run build",
```

Comment:
Hmm, the `webpack` step is in a `prebuild` script... why does that run even though it's not being called?

Reply:
Heh, I probably shouldn't bore you with all of the details of the lifecycles of npm scripts, but `pre` scripts get run before the script that shares a name with it; `prebuild` -> `build`. It's actually pretty handy for cases like this, where subo now only needs to worry about running the `build` command.